docs/articles/documentation/using-testcafe/command-line-interface.md

```diff
@@ -181,7 +181,7 @@ For `cmd.exe`:
 testcafe "path:`C:\Program Files (x86)\Google\Chrome\Application\chrome.exe` --start-fullscreen" tests/sample-fixture.js
 ```
 
-You can specify arguments for local browsers only.
+Only installed and portable browsers located on the current machine can be launched with arguments.
```

Comment:
I have doubts regarding this phrase. Does it sound awkward?

Reply:
For me it sounds good